docs/spec/functional_spec.rst

```diff
@@ -244,46 +259,146 @@ Liquidation Auctions
 --------------------------------------------
 
+Internal. Receive a liquidation slice (tez) from a burrow.
+
 ``receive_slice_from_burrow: (pair address nat)``
 
++---------------+-----------------------+-------------------------------------------------------------------------+
+| Parameter     |      Field Type       | Description                                                             |
++===============+=======================+=========================================================================+
+| owner         | address               | The burrow owner's address                                              |
++---------------+-----------------------+-------------------------------------------------------------------------+
+| id            | nat                   | The caller's ID for the burrow sending the slice                        |
++---------------+-----------------------+-------------------------------------------------------------------------+
+
+
 Maintenance entrypoints
 =======================
 
 Perform Checker internal maintenance
 ------------------------------------
 
+Perform housekeeping tasks on the contract state. This includes:
+
+#. updating the system parameters;
+#. accruing burrowing fees to the cfmm;
+#. updating auction-related info (completing an old / starting a new auction);
+#. processing a limited number of liquidation slices from completed auctions;
+#. updating the index by consulting the oracle.
+
+This operation credits an amount of kit (that is a function of time passed
+since the last time ``touch`` was called) to ``Tezos.sender``.
+
 ``touch: unit``
 
++---------------+-----------------------+-------------------------------------------------------------------------+
+| Parameter     |      Field Type       | Description                                                             |
++===============+=======================+=========================================================================+
+| unit          | unit                  | ()                                                                      |
++---------------+-----------------------+-------------------------------------------------------------------------+
+
+
 Apply an Oracle update
 ----------------------
 
+Internal. Receive a price update from the registered oracle.
+
 ``receive_price: nat``
 
++---------------+-----------------------+-------------------------------------------------------------------------+
+| Parameter     |      Field Type       | Description                                                             |
++===============+=======================+=========================================================================+
+| price         | nat                   | The current index, as a fixedpoint with a scaling factor of 1000000     |
++---------------+-----------------------+-------------------------------------------------------------------------+
+
+
 FA1.2 Interface
```

Comment:
Not about this PR, but this should probably be called "FA2 Interface".

```suggestion
FA2 Interface
```

Reply:
Ah, yes, I'll fix that, good catch :+1: